source/common/upstream/health_discovery_service.h

```diff
@@ -60,6 +114,16 @@ class HdsDelegate
   envoy::service::discovery::v2::HealthCheckRequest health_check_request_;
   std::unique_ptr<envoy::service::discovery::v2::HealthCheckSpecifier> health_check_message_;
   std::vector<std::string> clusters_;
+  Runtime::Loader& runtime_;
+  Envoy::Stats::Store& store_stats;
+  Ssl::ContextManager& ssl_context_manager_;
+  Secret::SecretManager& secret_manager_;
+  Runtime::RandomGenerator& random_;
+  Event::Dispatcher& dispatcher_;
+  Upstream::HealthCheckerSharedPtr health_checker_ptr;
+  envoy::api::v2::Cluster cluster_config;
```

Comment:
Why is there both `cluster_config` and `cluster_config_` here?

Reply:
There is now only one.